bugbug/bug_features.py

```diff
@@ -182,6 +182,10 @@ def __call__(self, bug):
 
 
 def cleanup_url(text):
+    mozilla_text = ['hg.mozilla', 'searchfox', 'dxr.mozilla']
+    for mozilla_keyword in mozilla_text:
+        if mozilla_keyword in text:
+            text = re.sub(r'https://{0}\S+|http://{0}\S+'.format(mozilla_keyword), 'CODE_REFERENCE_URL', text)
```

Comment:
Instead of applying multiple regexes, one for each URL, you should be able to have just one regex.
Also, you can make the `s` optional to avoid having the `|` between the https case and the http case.

Reply:
We have two tokens to use: URL and CODE_REFERENCE_URL. How can I use only one regex to make appropriate substitution?

Reply:
I mean one regex for CODE_REFERENCE_URL, currently you're making one per loop iteration (so three)

Reply:
Ah, got it.